src/Core/DatabaselessKernel.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace SilverStripe\Core;
+
+use Exception;
+
+/**
+ * Boot a kernel without requiring a database connection.
+ * This is a workaround for the lack of composition in the boot stages
+ * of CoreKernel, as well as for the framework's misguided assumptions
+ * around the availability of a database for every execution path.
+ *
+ * @internal
+ */
+class DatabaselessKernel extends BaseKernel
+{
+    protected $queryErrorMessage = 'Booted with DatabaseLessKernel, cannot execute query: %s';
```

Comment:
```suggestion
    private $queryErrorMessage = 'Booted with DatabaseLessKernel, cannot execute query: %s';
```

Reply:
What's this even used for?  It can probably just be removed

Reply:
Done